tools/setup-macos-ninja-debug.sh

```
@@ -4,19 +4,19 @@ script_dir="$(cd "$(dirname "${script_source}")" && pwd)"
-export conan_profile=${2:-scwx-macos_clang-18_armv8}
+export conan_profile=${2:-scwx-macos_clang-22_armv8}
 export generator=Ninja
 export qt_base="/Users/${USER}/Qt"
 export qt_arch=macos
 export address_sanitizer=${4:-disabled}
 
 # Set explicit compiler paths
-export CC=$(brew --prefix llvm@18)/bin/clang
-export CXX=$(brew --prefix llvm@18)/bin/clang++
-export PATH="$(brew --prefix llvm@18)/bin:$PATH"
+export CC=$(brew --prefix llvm@22)/bin/clang
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)